ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/xlc_v16/tasks/main.yml

```diff
@@ -2,47 +2,59 @@
 # IBM XL C 16.1.0 #
 ###################
 ---
-- name: Checking for XLC16 availability
+- name: Checking for xlC16 installed as 'licensed' LPP
   stat:
-    path: /opt/IBM/xlc/16.1.0/bin/xlc
+    path: /usr/lpp/xlCcmp.16.1.0.license
   register: xlc16
   tags: xlc16
 
-- debug:
+- name: Announce skip of download and installation
+  debug:
     msg: "xlc16 installed, skipping download and installation"
-  when: xlc16.stat.islnk is defined
+  when: xlc16.stat.isdir is defined and xlc.stat.isdir
```

Comment:
The error above is that one was xlc16.stat and the other a typo (xlc.stat).
I leave it double (isdefined, isTrue) because during my experiments with stat: I frequently ran into something such as xyz.stat.isdir failing because it was not defined - and that is not the same as false. -- correcting the typo